plugins/channeltx/moddatv/dvb-s2/DVB2.cpp

```diff
@@ -273,8 +273,6 @@ int DVB2::next_ts_frame_base( u8 *ts )
 }
 DVB2::DVB2(void)
 {
-    // Clear the transport queue
-    m_tp_q.empty();
```

Comment:
m_tp_q appears to be unused so it can be removed from the class.

Reply:
It's used in "unpack_transport_packet_add_crc" as a temporary storage. But I think it's kept inside class to (maybe?) reuse some internal structures.